src/sidebar/index.js

```diff
@@ -1,10 +1,10 @@
-const addAnalytics = require('./ga');
-const disableOpenerForExternalLinks = require('./util/disable-opener-for-external-links');
-const { fetchConfig } = require('./util/fetch-config');
-const serviceConfig = require('./service-config');
-const { jsonConfigsFrom } = require('../shared/settings');
-const crossOriginRPC = require('./cross-origin-rpc.js');
-const sentry = require('./util/sentry');
+import addAnalytics from './ga';
```

Comment:
I note this diff doesn't address ordering, which is fine as it cuts down on distractions, but these imports perhaps could use it at some point...